Conduit/Conduit.Db/ConduitContext.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Conduit.Db
+{
+    public  class ConduitContext: DbContext
+    {
+        public ConduitContext(DbContextOptions<ConduitContext> options) : base(options)
+        {
+
+        }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
```

Comment:
@noor848 do you create the DB from SQL files? if yes , can you attached them in folder please?
if No, you can here declare the key for each table (primary/foregin)